modules/gitops-logging-into-keycloak.adoc

```diff
@@ -0,0 +1,66 @@
+[id="logging-into-keycloak_{context}"]
+= Logging into Keycloak
+
+Log in to the Keycloak console after configuring the Argo CD instance to manage identities or roles and access provided to the various roles. To log in, first you have to get the Keycloak admin username and password.
+
+.Procedure
+
+. Get the Keycloak Route URL for Login:
++
+[source,yaml]
+----
+$ oc -n argocd get route keycloak
+
+NAME        HOST/PORT                                                        PATH   SERVICES   PORT    TERMINATION   WILDCARD
+keycloak    keycloak-default.apps.ci-ln-******.origin-ci-int-aws.dev.**.com         keycloak   <all>    reencrypt     None
+----
+. Get the Keycloak pod name which stores the username and password as environment variables:
```

Comment:
```suggestion
. Get the Keycloak pod name that stores the user name and password as environment variables:
```
Per the IBM SG: 'Use "that", without a comma, to introduce a restrictive clause. Use "which", preceded by a comma, to introduce a nonrestrictive clause.'